pubsub/src/main/java/com/github/twitch4j/pubsub/events/PinnedChatCreateEvent.java

```diff
@@ -0,0 +1,13 @@
+package com.github.twitch4j.pubsub.events;
+
+import com.github.twitch4j.common.events.TwitchEvent;
+import com.github.twitch4j.pubsub.domain.PinnedChatData;
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+
+@Value
+@EqualsAndHashCode(callSuper = false)
+public class PinnedChatCreateEvent extends TwitchEvent {
```

Comment:
Should the past participle form of the verb be preferred for these event names?

Reply:
sounds good, `Created`, `Updated` and `Deleted` for the three events.

Reply:
Fixed in https://github.com/twitch4j/twitch4j/pull/666/commits/1cb5e9b3a0564f3d8141b1f4538bad420cc26513